src/test/java/com/rcv/TabulatorTests.java

```diff
@@ -126,10 +126,14 @@ private static void runTabulationTest(String stem) {
     // compare actual to expected
     assertTrue(fileCompare(expectedPath, actualSummaryOutputPath));
     // test passed so cleanup test output folder
-    File ouputFolder = new File(session.outputPath);
-    for (File file : ouputFolder.listFiles()) {
-      if (!file.isDirectory()) {
-        file.delete();
+    File outputFolder = new File(session.outputPath);
+    if (outputFolder.listFiles() != null) {
+      //noinspection ConstantConditions
+      for (File file : outputFolder.listFiles()) {
+        if (!file.isDirectory()) {
+          //noinspection ResultOfMethodCallIgnored
```

Comment:
Is this some weird IntelliJ thing? Seems kind of overboard to have to add this kind of comment every time you don't want to use a method's return value...

Reply:
Yes it is. Luckily it only occurs in this one spot. FWIW our code base at work is rife with these sorts of suppression statements.